manager/controllers/utils.go

```diff
@@ -74,12 +73,7 @@ func ApplySort(c *gin.Context, tx *gorm.DB, fieldExprs database.AttrMap,
 
 	// We sort by a column expression and not the column name. The column expression is retrieved from fieldExprs
 	for _, enteredField := range fields {
-		origEnteredField := enteredField // needed for showing correct info in `meta` section
-		if apiver < 3 && strings.Contains(enteredField, "applicable_systems") {
-			// `applicable_systems` is used in v1 and v2 API for consistency, in fact it is installable_systems
-			// therefore, if user wants to sort by `applicable_systems`, we need to sort `installable_systems` in DB
-			enteredField = strings.Replace(enteredField, "applicable", "installable", 1)
-		}
+		origEnteredField := enteredField                                               // needed for showing correct info in `meta` section
```

Comment:
some more tabs were added and they are the reason why you had to add nolint:lll on line 60, please remove those tabs and nolint ignore for lll

Reply:
it's done by gofmt

Reply:
it's caused by gofmt